bot/logger/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Module for Logging functionality.
+It exposes log and command_log.
+
+'command_log' is a decorator to be used on command functions. It works on any function whose first argument is a
+context object. It logs when a command is called by a user. Additional information include the calling user and channel.
+
+log is a logger object used to log information on various levels in the code. The logs will be written to the stderr and
```

Comment:
'*log'

Reply:
*log?

Reply:
*`log`

Reply:
I am still as confused as before?
You want me to make ' ' at the log word at the beginning of the sentence? 
Or I made a typo? Or you want me to add the word somewhere?